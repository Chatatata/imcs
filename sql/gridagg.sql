select cs_grid_max(Close,2) from Quote_get('IBM');
select cs_grid_min(Close,2) from Quote_get('IBM');
select cs_grid_sum(Close,2) from Quote_get('IBM');
select cs_grid_avg(Close,2) from Quote_get('IBM');
select cs_grid_var(Close,2) from Quote_get('IBM');
select cs_grid_dev(Close,2) from Quote_get('IBM');
