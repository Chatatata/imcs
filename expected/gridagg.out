select cs_grid_max(Close,2) from Quote_get('IBM');
       cs_grid_max       
-------------------------
 float4:{20.2,40.2,50.5}
(1 row)

select cs_grid_min(Close,2) from Quote_get('IBM');
       cs_grid_min       
-------------------------
 float4:{10.5,30.2,50.5}
(1 row)

select cs_grid_sum(Close,2) from Quote_get('IBM');
       cs_grid_sum       
-------------------------
 float8:{30.7,70.4,50.5}
(1 row)

select cs_grid_avg(Close,2) from Quote_get('IBM');
       cs_grid_avg        
--------------------------
 float8:{15.35,35.2,50.5}
(1 row)

select cs_grid_var(Close,2) from Quote_get('IBM');
      cs_grid_var      
-----------------------
 float8:{23.5225,25,0}
(1 row)

select cs_grid_dev(Close,2) from Quote_get('IBM');
    cs_grid_dev    
-------------------
 float8:{4.85,5,0}
(1 row)

